### ISSUE TITLE
Saving the config-page causes a boot loop

### ISSUE THREAD

**Issue by Einstein2150** [exploitagency/ESPortalV2#2]
When saving the configuration i.e. only rename the access point and click on save it cause a bootloop. You can see flashing the two led on the board. Console prints permanent output (not readable).

Even flashing a new image doesn't change anything. SPIFFS must be erased manually so there must be an issue in the write-function.

**Comment by exploitagency** [comment]
Are there any special symbols in access point name?

**Comment by Einstein2150** [comment]
No. Only the blank character (space). It would be helpful if there would be some serial output during the writing-process (Begin, Writing entry x y z, finished) to avoid a failure during saving the JSON to SPIFFS. Maybe there must be a integrity-check too before loading a (maybe) corrupted JSON-Config during start.

**Comment by akshayaruku** [comment]
Getting same loop
.No meta and special characters used

**Comment by odolezal** [comment]
I can confirm this issue.
